pom.xml

```diff
@@ -966,7 +966,7 @@
     <versions-maven-plugin.version>2.7</versions-maven-plugin.version>
     <xml-maven-plugin.version>1.0.2</xml-maven-plugin.version>
     <maven-license-plugin.version>1.7</maven-license-plugin.version>
-    <incrementals-plugin.version>1.0-beta-5</incrementals-plugin.version>
+    <incrementals-plugin.version>1.1</incrementals-plugin.version>
     <incrementals-enforce-minimum.version>1.0-beta-4</incrementals-enforce-minimum.version>
```

Comment:
@jglick should we update it as well after Beta?

Reply:
Not necessary.